articles/quickstart/webapp/laravel/01-login.md

```diff
@@ -56,41 +56,59 @@ First, we need to add the Auth0 Services to the list of Providers in `config/app
 ```php
 // config/app.php
 
-'providers' => array(
-    // ...
-    Auth0\Login\LoginServiceProvider::class,
-);
+<?php
+
+    return [
+        // ...
+
+      'providers' => [
+          // ...
+          Auth0\Login\LoginServiceProvider::class,
+      ],
 ```
 
 If you want to use an `Auth0` facade, add an alias in the same file (not required, [more information on facades here](http://laravel.com/docs/8.x/facades)):
 
 ```php
 // config/app.php
 
-'aliases' => [
-    // ...
-    'Auth0' => Auth0\Login\Facade\Auth0::class,
-];
+<?php
+
+    return [
+        // ...
+
+      'aliases' => [
+          // ...
+          'Auth0' => Auth0\Login\Facade\Auth0::class,
+      ],
```

Comment:
Is it expected that we are:

- opening the php tag but not closing it ?
- opening an array but not closing it ?

Reply:
PHP tags are never closed in files that are entirely PHP, that's alright.

This section of the quickstart is suggesting readers 'add an alias in the same file' and not replace the contents of that file with content, so we don't need to rewrite all of the code in this portion of the QS or make it fully syntactically accurate, I think. We're just indicating the relevant milestones in the code so readers can find the position of the position in the file so readers can find it.

Reply:
> PHP tags are never closed in files that are entirely PHP, that's alright.

TIL! Thanks for explaining!